marshal.go

```diff
@@ -81,7 +81,7 @@ func Marshal(info TypeInfo, value interface{}) ([]byte, error) {
 		return marshalDouble(info, value)
 	case TypeDecimal:
 		return marshalDecimal(info, value)
-	case TypeTimestamp:
+	case TypeTimestamp, TypeTime:
 		return marshalTimestamp(info, value)
```

Comment:
The semantics of [`timestamp`](http://cassandra.apache.org/doc/latest/cql/types.html#working-with-timestamps) and [`time`](http://cassandra.apache.org/doc/latest/cql/types.html#working-with-times) are different. AFAICT, `marshalTimestamp()` will marshal `time.Time` and `time.Duration` incorrectly for `time`.